docs/en/04_Changelogs/4.11.0.md

```diff
@@ -78,6 +79,15 @@ The version portion of the metagenerator tag can be disabled via:
 SilverStripe\CMS\Model\SiteTree:
   show_meta_generator_version: false
 ```
+### Allow-plugins configuration option in Composer versions 2.2.0 and up
```

Comment:
```suggestion
### Allow-plugins configuration option in Composer versions 2.2.0 and up {#composer}
```
Sorry, didn't notice this the first time around. This is needed so that the link above anchors to this heading correctly.